### PR TITLE
Safari and Firefox `impl_url` for the Web Bluetooth API and bluetooth web-features tag

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary

This PR links the various Web Bluetooth API features to the corresponding tracking bug on the Safari WebKit tracker. That bug was closed as WONTFIX because Apple is not considering implementing the feature. But it's still the best URL we have to link developers to, I think. That bug has many comments from 3rd-party developers asking for the feature in Safari. So I think this is the right place to link to.

This PR also adds the `web-features:bluetooth` tag to all the Web Bluetooth API BCD entries, so that we can define a new feature over on the web-features repo.

### CODE REVIEW

**Comment by captainbrosset** [comment]
> I think if we link to the webkit wontfix bug, we should also link to the gecko wontfix bug: https://bugzilla.mozilla.org/show_bug.cgi?id=674737

Sure, the more the better! Thanks for the bug link. Adding it now.

File: api/BluetoothUUID.json [hunk 1]
```diff
@@ -186,14 +206,18 @@
           "description": "<code>getService()</code> static method",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/BluetoothUUID/getService_static",
           "spec_url": "https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothuuid-getservice",
+          "tags": [
+            "web-features:bluetooth"
```
Comment (Elchi3) [inline api/BluetoothUUID.json:210]:
```suggestion
            "web-features:web-bluetooth"
```

File: api/Navigator.json [hunk 2]
```diff
@@ -340,6 +340,9 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/bluetooth",
           "spec_url": "https://webbluetoothcg.github.io/web-bluetooth/#dom-navigator-bluetooth",
+          "tags": [
+            "web-features:bluetooth"
```
Comment (Elchi3) [inline api/Navigator.json:344]:
```suggestion
            "web-features:web-bluetooth"
```

**Review by Elchi3 — APPROVED** [review]
Thanks Patrick! 👍

**Pull request merged**